app/main.py

```diff
@@ -1,8 +1,23 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
 
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        Person.people[name] = self
 
-def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+
+def create_person_list(people: list) -> None:
+    person_list = []
+    for person in people:
+        person_instance = Person(name=person["name"], age=person["age"])
+        person_list.append(person_instance)
```

Comment:
Consider using list comprehension to create the `person_list`. This aligns with the checklist recommendation to use list comprehension when creating a list of instances from a collection.